app/javascript/pages/Event/index.js

```diff
@@ -148,8 +163,8 @@ const Event = ({ data: { loading, event, currentUser }, createSignup, destroySig
             </Section>
           </div>
           <div className={s.rightCol}>
-            <Section title="Information">
```

Comment:
Is this never populated?

Reply:
Yes, as far as I can see, thus I repurposed it for "Add to Google Calendar" button.